Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -665,9 +678,18 @@ void AzAssetBrowserWindow::SetTwoColumnMode(QWidget* viewToShow)
 
 void AzAssetBrowserWindow::SetOneColumnMode()
 {
-    m_ui->m_middleStackWidget->hide();
-    m_ui->m_searchWidget->RemoveFolderFilter();
-    m_ui->m_thumbnailView->SetThumbnailActiveView(false);
+    if (m_ui->m_thumbnailView->GetThumbnailActiveView() || m_ui->m_expandedTableView->GetExpandedTableViewActive())
+    {
+        m_ui->m_middleStackWidget->hide();
+        m_ui->m_assetBrowserTreeViewWidget->SetApplySnapshot(false);
+        m_ui->m_searchWidget->RemoveFolderFilter();
+        if (!m_ui->m_assetBrowserTreeViewWidget->selectionModel()->selectedRows().isEmpty())
+        {
+            m_ui->m_assetBrowserTreeViewWidget->expand(m_ui->m_assetBrowserTreeViewWidget->selectionModel()->selectedRows()[0]);
+        }
```

Comment:
Can there be multiple selected rows? If so, should we expand each of those indexes?

Reply:
There can be, but it seems to me that having a selected row in the tree view doesn't necessarily mean the user wants all those folders expanded if they switch views? 
I was thinking that it makes sense to expand the current folder that the user is viewing, but not all of them since the multi-select might be for folder operations